src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/HtmlTag.java

```diff
@@ -687,7 +666,8 @@ public Attr getAttr(Name attrName) {
     }
 
     public AttrKind getAttrKind(Name attrName) {
-        AttrKind k = attrs.get(getAttr(attrName)); // null-safe
+        Attr attr= getAttr(attrName);
+        AttrKind k = attr.isGlobal()? AttrKind.OK: attrs.get(attr); // null-safe
```

Comment:
As hannes suggested, `getAttrKind` now returns `AttrKind.OK` if we are dealing with a global attribute.